src/main/webapp/app/modules/canvas/filter/filter-element.tsx

```diff
@@ -123,12 +124,7 @@ const FilterElement = (props: IFilterElementProp) => {
   };
 
   function removeFilter(filter) {
-    let filterParameters;
-    filterParameters = props.filters;
-    filterParameters[filter] = [];
-    props.saveFilter(filterParameters);
-
-    filterParameters = props.selectedFilter;
+    const filterParameters = props.selectedFilter;
```

Comment:
no need to assign into new var. pass selected filter directly

Reply:
done